### PR TITLE
BZ-1920414: Clarifying the revision numbers are examples

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://bugzilla.redhat.com/show_bug.cgi?id=1920414

Preview: 

* https://deploy-preview-30077--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html
* https://deploy-preview-30077--osdocs.netlify.app/openshift-enterprise/latest/security/audit-log-policy-config.html#configuring-audit-policy_audit-log-policy-config

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@retroflexer @geliu2016 Can you please review this update? It's to make it clearer that the revision number didn't have to be 3, that 3 was just the revision in this example. I changed it to 7, so they didn't think it had to match the number of nodes or anything either.

Preview: https://deploy-preview-30077--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html

File: modules/dr-restoring-cluster-state.adoc [hunk 1]
```diff
@@ -211,10 +211,11 @@ Review the `NodeInstallerProgressing` status condition for etcd to verify that a
 [source,terminal]
 ----
 AllNodesAtLatestRevision
-3 nodes are at revision 3
+3 nodes are at revision 7 <1>
 ----
+<1> Confirm that this message states that all nodes are at the latest revision number, which in this example is 7.
```
Comment (kalexand-rh) [inline modules/dr-restoring-cluster-state.adoc:216]: We don't usually give instructions in callouts, so I might say:
```suggestion
<1> In this example, the latest revision number is `7`.
```

File: modules/dr-restoring-cluster-state.adoc [hunk 2]
```diff
@@ -211,10 +211,11 @@ Review the `NodeInstallerProgressing` status condition for etcd to verify that a
...
 +
-If the output shows a message such as `2 nodes are at revision 3; 1 nodes are at revision 4`, this means that the update is still in progress. Wait a few minutes and try again.
+If the output shows a message such as `2 nodes are at revision 6; 1 nodes are at revision 7`, this means that the update is still in progress. Wait a few minutes and try again.
```
Comment (kalexand-rh) [inline modules/dr-restoring-cluster-state.adoc:218]: Maybe
```suggestion
If the output includes multiple revision numbers, such as `2 nodes are at revision 6; 1 nodes are at revision 7`, this means that the update is still in progress. Wait a few minutes and try again.
```

**Review by kalexand-rh — COMMENTED** [review]
I have a couple of suggestions, but this change is looking good!

File: modules/dr-restoring-cluster-state.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/dr-restoring-cluster-state.adoc:216]: Good point. I'll adjust

**Comment by geliu2016** [comment]
LGTM, thanks

**Pull request merged**